nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -88,32 +88,32 @@ in {
 
     enableBrokenCiphersForSSE = mkOption {
       type = types.bool;
-      # Workaround can be removed at backport-time for 22.11.
-      default = !(versionOlder stateVersion "22.11");
+      default = versionOlder stateVersion "22.11";
```

Comment:
Hm, shouldn't that be a not here?
I read it as: `if stateVersion > 22.11 then true else false`, meaning, broken ciphers for SSE are enabled for ≥ 23.05 and disabled for ≤ 22.11, which is the contrary of what we want. Did I miss something about the logic of that function?

Reply:
I read the opposite: `if stateVersion < 22.11 then true else false`

Older version means smaller version number, right ?

Reply:
Worked too much, today, I guess :D — ignore my comment please

Reply:
@drupol I guess the confusion stems from the ordering of the arguments (and I can totally related to that, I always have to think twice as well, otherwise I would've spotted it yesterday already ;-) )

`versionOlder` evaluates to `true` if the first argument is smaller (i.e. older) than the second one:

```
nix-repl> :l <nixpkgs/lib>
Added 402 variables.

nix-repl> stateVersion = "22.05"

nix-repl> versionOlder stateVersion "22.11"
true
```

And this is precisely what we want.